portal/models/app_text.py

```diff
@@ -120,14 +135,16 @@ class Terms_ATMA(AppTextModelAdapter):
 
     @staticmethod
     def name_key(**kwargs):
-        """Generate AppText name key for a Terms and Conditions - new item just added
-
-        Not expecting any args at this time - may specialize per study
-        or organization in the future as needed.
+        """Generate AppText name key for a Terms and Conditions
 
+        :param organization: optional, present in tandem to role parameter
+        :param role: optional, role of the user
         :returns: string for AppText.name field
 
         """
+        if kwargs.get('organization') and kwargs.get('role'):
```

Comment:
is it all or none?  should user be informed if they pass 'organization' and not 'role'?